docs/network-policy.md

```diff
@@ -229,7 +231,7 @@ to the 10.0.10.0/24 subnet specified by the `ipBlock` field.
 **Note**: The order in which the egress rules are set matter, i.e. rules will
 be enforced in the order in which they are written.
 
-### Behavior of `to` and `from` selectors
+### Behavior of to and from selectors
```

Comment:
We are not able to quote "to" and "from"?

Reply:
the tool replaces the quotes with <code> tags which then do not work as links.. so i removed the quotes.. do you think we should find an alternative to maintain the quotes?

Reply:
I think that limitation is ok and not worth spending too much time on. After a quick search it seems to be related to the markdown parsing library and its handling of inline code: https://github.com/gomarkdown/markdown. If no obvious workaround, I would suggest using quotes around `to` and `from` or alternatively `*` (italic format) to make them stand out. I don't know if these will play nicely with mdtoc.

Reply:
italic format does render proper.. updated with it